sheets/ssh

```diff
@@ -46,7 +46,7 @@ ssh -i some_id_rsa -o IdentitiesOnly=yes them@there:/path/
 # Temporarily disable `pubkey` authentication for this instance.
 ssh -o PubkeyAuthentication=no username@hostname.com
 
-# Mount a remote directory or filesystem, through SSH, to a local mountpoint.
+# Mount a remote directory or filesystem, through SSH, to a local mount point.
```

Comment:
This is another case of this, IMO, being fine enough; it's so often officially documented as a single word, such as in mount(8). That being said, I'd say you're technically correct here.